src/Component/Core/Admin/Resources/routing/product_listing.yaml

```
@@ -0,0 +1,32 @@
+open_marketplace_admin_product_listing_show:
+    path: /product-listings/{id}
+    defaults:
+        _controller: bitbag.open_marketplace.component.core.admin.controller.product_listing.show
+        _sylius:
+            section: admin
+            template: "Context/Admin/ProductListing/show.html.twig"
+            permission: true
+
+open_marketplace_admin_product_listing_accept:
+    path: /product-listings/{id}/accept
+    defaults:
+        _controller: bitbag.open_marketplace.component.core.admin.controller.product_listing.accept
+        _sylius:
+            section: admin
+            permission: true
+
+open_marketplace_admin_product_listing_reject:
+    path: /product-listings/{id}/reject
+    defaults:
+        _controller: bitbag.open_marketplace.component.core.admin.controller.product_listing.reject
+        _sylius:
+            section: admin
+            permission: true
+
+open_marketplace_admin_product_listing_restore:
+    path: /product-listings-restore/{id}
```

Make it an action resource like the other routes
```suggestion
    path: /product-listings/{id}/restore
```